integration_tests/create_token_test.go

```diff
@@ -82,6 +82,7 @@ func fetchCode(id, redirect, scope string, controller *controllers.OAuthControll
 	values.Add("scope", scope)
 	values.Add("login", testAccountLogin)
 	values.Add("password", testAccountPassword)
+	values.Add("expires_at", "3600")
```

Comment:
@im-adithya this should be a much larger time. The timestamp right now is `1692085297`